doc-content/enterprise-only/openshift/authoring-params.adoc

```diff
@@ -27,6 +27,9 @@ Alternatively, if you want to use the Maven repository that is built into {CENTR
 ** *Password for the Maven service hosted by {CENTRAL}* (`{CENTRAL_CAPITAL_UNDER}_MAVEN_PASSWORD`): The password for the built-in Maven repository.
 endif::DM[]
 +
+:params_offline_central:
+include::offline-params.adoc[]
+:params_offline_central!:
 . You can use Git hooks to facilitate interaction between the internal Git repository of {CENTRAL} and an external Git repository. To configure Git hooks, set the following parameter:
 ** *Git hooks directory* (`GIT_HOOKS_DIR`): The fully qualified path to a Git hooks directory, for example, `/opt/eap/standalone/data/kie/git/hooks`. You must provide the content of this directory and mount it at the specified path; for instructions, see <<githooks-proc>>.
```

Comment:
Suggest "... path. For instructions on / details about setting up Git hooks, ..." (or whatever).  I know you love the semicolon, but imo, it should really be reserved for two closely connected clauses where a comma isn't enough but a period is too much. Should be pretty rare. It's not a commonly understood punctuation mark in user communication.

Reply:
Separate problem - not this PR, but the ongoing holistic review. The problem here is that the correct wording would be "For instructions about providing and mounting this directory, see..." - a huge repeat. This is the problem with most of my semicolons. I understand there is a consistency issue but don't want to hold this particular PR back for it, as there will be another tomorrow that is more relevant.